skbuild/setuptools_wrap.py

```diff
@@ -59,19 +63,21 @@ def parse_args():
                              concatenate_value=True)
     dutils, cmake = move_arg('-G', dutils, cmake)
     dutils, make = move_arg('-j', dutils, make)
-    op = os.path
-
-    def absappend(x):
-        return op.join(op.dirname(op.abspath(sys.argv[0])), x)
+    absappend = (
+        lambda x: os.path.join(
```

Comment:
Do not use lambda here. See https://github.com/scikit-build/scikit-build/commit/1cd88e7b24cfe03fb06472de8a6619cc38f96439